# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../kdegraphics/kuickshow-imlib-libs.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- kdegraphics-3.5.2/kuickshow/src/Makefile.am.orig	2005-09-10 10:19:04.000000000 +0200
+++ kdegraphics-3.5.2/kuickshow/src/Makefile.am	2006-04-18 21:52:54.000000000 +0200
@@ -7,6 +7,7 @@
 # set the include path for X, qt and KDE
 INCLUDES = $(all_includes)
 KDE_CXXFLAGS = $(IMLIB_CFLAGS)
+LIBS=$(shell imlib-config --libs) -lm
 METASOURCES = AUTO
 
 kuickshow_la_LDFLAGS = $(all_libraries) -module -avoid-version
